saleor/graphql/translations/mutations.py

```diff
@@ -60,6 +62,24 @@
 }
 
 
+def validate_input(model: Model, input_data: dict):
+    errors = {}
+    for field_name, value in input_data.items():
+        model_field = model._meta.get_field(field_name)
+        if not isinstance(model_field, CharField):
+            continue
+        if value:
+            print(field_name, len(value), model_field.max_length)
+        if value and len(value) > model_field.max_length:
+            errors[field_name] = ValidationError(
+                f"This value can't be longer than {model_field.max_length}",
```

Comment:
Missing `.`?

Reply:
Maybe? Plenty of validation error messages returned by API don't include dot at the end.

Reply:
Maybe.